components/button/button-subtle.js

```diff
@@ -182,6 +185,7 @@ class ButtonSubtle extends ButtonMixin(RtlMixin(LitElement)) {
 				<span class="d2l-button-subtle-content">${this.text}</span>
 				<slot></slot>
 			</button>
+			${this.description ? html`<span id="${this._describedById}" hidden>${this.description}</span>` : null}
```

Comment:
I'm using `hidden` here instead of offscreen as @dbatiste noticed that hidden elements can act in this way, which is pretty sweet. Tested this in NVDA and Voiceover.